### PR TITLE
updated adding/removing item to cart/wishlist

### PR DESCRIPTION
**Pull request by vidhi-thakur**
updated the functionality to remove or add item from cart and wishlist such that if the same item is added again, the count increases instead of adding the redundant item

P.S.: Please suggest if there is a better way of implementing add or remove functionality.

### CODE REVIEW

File: src/reducers/cartReducer.js [hunk 1]
```diff
@@ -1,15 +1,33 @@
 export const initialCart = {
     cartItems: [],
-    totalAmount: 0
+    totalAmount: 0,
+    numOfItemsinCart: 0
 }
 
-function cartReducer(state, {type, payload}) {
+function cartReducer(state, { type, payload }) {
+    const itemPresent = state.cartItems.filter(item => item.id === payload.id)
```
Comment (msdeshmukh009) [inline src/reducers/cartReducer.js:8]: You can move this logic to the component itself and try to have two more actions like `INCREAMENT_QNT` & `DECREMENT_QNT` which will handle your quantity-related operations.

**Review by msdeshmukh009 — APPROVED** [review]
The cart management is working fine. 
Suggestion: Instead of increasing the quantity in case of wishlist, you can remove the existing item from the wishlist.  And also I think it is unnecessary to increase the badge count when the quantity is increased.

**Review by kumsomi — APPROVED** [review]
- The UI looks good
- Functionality is working fine
- Hope you'll add increment and decrement the quantity options in add to cart and add to wishlist option

File: src/reducers/cartReducer.js (hunk 1 above)
Comment (vidhi-thakur) [reply src/reducers/cartReducer.js:8]: cool, will update that

**Pull request merged**